spec/services/message_employee_matcher_spec.rb

```diff
@@ -18,6 +18,26 @@
       Timecop.return
     end
 
+    it "matches messages to users if the user's time has advanced past the scheduled message send at time but hasn't been sent yet" do
+      Timecop.freeze(Time.current)
+
+      days_after_start = 3
+      scheduled_message_time = Time.current.in_time_zone("Eastern Time (US & Canada)")
+      scheduled_message_time -= 1800
```

Comment:
Ah good point, addressing that - it's meant to represent a difference of half an hour (so simulating the scheduled message being set to be sent in the past from the current time the test is run).